targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_VK_RZ_A1H/device.h

```diff
@@ -31,6 +31,7 @@
 
 
 
+#define TRANSACTION_QUEUE_SIZE_SPI 16
```

Comment:
Can you delete the unnecessary empty lines.

Reply:
Done.